src/jbod/disks.rs

```diff
@@ -209,13 +209,10 @@ pub mod DiskShelf {
             let stdout_lines = stdout_reader.lines();
 
             for line in stdout_lines {
-                let line_unwrap = line.unwrap().to_owned();
+                let line_unwrap = line.unwrap();
                 let output_split: Vec<&str> = line_unwrap.split_whitespace().collect();
-                if output_split.len() > 1 {
-                    disks.insert(output_split[0].to_string(), output_split[1].to_string());
-                } else {
-                    disks.insert(output_split[0].to_string(), "NONE".to_string());
-                }
+                let val = output_split.get(1).unwrap_or(&"NONE");
```

Comment:
[Vec::get](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.get) and [Vec::get_mut](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.get_mut) are great, rustc generate code without panic-bound-check-error code with it.

Code end up quite well optimized and it's quite readable.

The whole futue API who overs `get` and `get_mut` is https://doc.rust-lang.org/std/slice/trait.SliceIndex.html but for now it's only stable on `Vec<T>`.